src/core/public/http/types.ts

```diff
@@ -205,12 +205,7 @@ export interface HttpRequestInit {
 
 /** @public */
 export interface HttpFetchQuery {
-  [key: string]:
-    | string
-    | number
-    | boolean
-    | undefined
-    | Array<string | number | boolean | undefined>;
+  [key: string]: string | number | boolean | string[] | number[] | boolean[] | undefined | null;
```

Comment:
nit: can we not change the format, put it one line is hard to debug.

Reply:
Yeah I can make that change right now